webpack.config.js

```diff
@@ -205,6 +205,12 @@ function devServerProxyBypass({ path }) {
     /* eslint-disable array-callback-return, indent */
 
     if ((Array.isArray(configs) ? configs : Array(configs)).some(c => {
+                if (path === '/') {
```

Comment:
I understand this is here for testing, but I don't think we'll want to keep this in this form.

Reply:
What do you mean by form? Also, I was planning on removing these files from the PR when we we everything else was checked off on as this is just a work around until the feature flag is updated.